about.html

```diff
@@ -13,56 +13,75 @@
 
 <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
+             <header>
+            <img class = "lambdaBlack" src = C:\Users\Mitch\Desktop\web-Sprint-Challenge-User-Interface-and-Git\img\lambda-black.png>
+            <nav>
+                <a href="index.html">Home</a>
+                <a href="about.html">About</a>
+                <a href="#">Products</a>
+                <a href="#">Blog</a>
+                <a href="#">Contact</a>
+            </nav>
+        </header>
+        <img src = C:\Users\Mitch\Desktop\web-Sprint-Challenge-User-Interface-and-Git\img\jumbo-about.png>
```

Comment:
There are some issues with the way you are importing your images. This works on your local computer but not others because you are using the Windows Path, which includes your Windows Username. since about.html is in the root directory of your project you could import the image by simply "/img/jumbo-about.png" <--- this is shorter and it uses a much simpler path. As you progress through Lambda School this will become second nature